lib/page/login/login_page.dart

```diff
@@ -41,8 +42,7 @@ class _LoginPageState extends ConsumerState<LoginPage> {
     ) {
       newLoginState.maybeWhen(
         success: () => _navigateToHome(),
-        apiError: (error) =>
-            _showError(AppLocalizations.of(context)!.loginError),
+        apiError: () => _showError(AppLocalizations.of(context)!.loginError),
```

Comment:
same, should we show the error message from error object first, if not, we can show the loginError string 🤔